pagopa/io-web-profile
# [IOPID-1215] : "Informativa Privacy" Style Adjustment

## Short description

The purpose of this PR is to align the style of the policy page more closely with the other products already created. 
Margins of containers have been adjusted, and titles have been made more prominent as [requested](https://pagopa.atlassian.net/browse/IOPID-1215)

The sample used for reference was that of [SEND](https://notifichedigitali.pagopa.it/informativa-privacy/)

## How to test
Set `NEXT_PUBLIC_DEV_MODE=true` the local env file.
Launch the app.
From the footer, access the 'Informativa Privacy' page
Verify the style
